samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_shell.c

```diff
@@ -418,8 +419,9 @@ static int dect_phy_mac_dissociate_cmd(const struct shell *shell, size_t argc, c
 
 	ret = dect_phy_mac_ctrl_dissociate(&params);
 	if (ret) {
-		desh_error("Cannot send association to FT %u's random access resource, err %d",
-			   params.target_long_rd_id, ret);
+		desh_error("Cannot send association release to FT %u's random access resource, "
+			   "err %d",
+				params.target_long_rd_id, ret);
```

Comment:
Is indentation fine in real life and the view makes it look weird?

Reply:
For me it looks ok in code.